content/admin/org_settings/API-credentials.md

```diff
@@ -0,0 +1,27 @@
+<!--
+title: "Security API Keys"
+description: "About article for the API Keys"
+tags: "Admin system settings security API keys configuration"
+-->
+
+When you download the Contrast JVM plugin (also called the Engine), it comes pre-fitted with a set of randomly generated credentials for your organization that don't involve the passwords of any users in the organization. When the plugin communicates with the Contrast application, it authenticates using these credentials. 
```

Comment:
Not sure if we still want to call it "Engine". In the TeamServer UI, we changed all the labels/texts from Engine to Agent